Makefile

```diff
@@ -96,7 +96,7 @@ test_images:
 	${IMG_BUILDER} build -t mpioperator/mpi-pi:openmpi examples/v2beta1/pi
 	${IMG_BUILDER} build -t mpioperator/intel build/base -f build/base/intel.Dockerfile
 	${IMG_BUILDER} build -t mpioperator/intel-builder build/base -f build/base/intel-builder.Dockerfile
-	${IMG_BUILDER} build -t mpioperator/mpi-pi:intel examples/v2beta1/pi -f examples/pi/intel.Dockerfile
+	${IMG_BUILDER} build -t mpioperator/mpi-pi:intel build/base -f build/base/intel.Dockerfile
```

Comment:
```suggestion
	${IMG_BUILDER} build -t examples/v2beta1/pi -f examples/v2beta1/pi/intel.Dockerfile
```

Reply:
good we have E2E tests :)

Reply:
tag the image as `-t examples/v2beta1/pi` ?

Reply:
Oops, deleted too much

Reply:
You applied the old edit :(